assets/dev/scss/editor/preview/_container.scss

```diff
@@ -181,8 +181,8 @@
 		}
 	}
 
-	// To avoid horizontal scroll in `flex-direction: column`
-	&.e-con--column {
+	// To avoid horizontal scroll
+	&.e-con {
```

Comment:
@ArmanPurtoian This now targets `<div class="e-con e-con">` because of the `&`. Was this your intention?
Otherwise you can just move the CSS up in the file under `.e-con {}`?